### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by sleepypioneer**
Adds a note about this being a fork (maybe redundant if you reconnect it @pbabbicola) and where the API token is stored for publishing orbs as well as trouble shooting.

### CODE REVIEW

**Comment by sleepypioneer** [comment]
Failing because of the Xcode macros, this is updated in #8 so ill merge in main after that is merged.

**Review by Zissi — APPROVED** [review]

**Pull request merged**
